recipes/brotli/test_brotli.py

```diff
@@ -4,6 +4,5 @@ def test_basic():
     plain = b"it was the best of times, it was the worst of times"
     compressed = brotli.compress(plain)
 
-    compressed = compressed.hex()
```

Comment:
This sees like an odd removal - what's the reason for this?

Reply:
The test didn't work....and it didn't really make a lotta sense to me.

If you consider the next assert:
```python
assert len(compressed) < len(plain)
```
This is testing the number of hexadecimal numbers for the compressed data in hex notation is less than the number of bytes in the plaintext. Converting bytes to hex and then representing that in 8-bit ASCII doubles the number of bytes to represent data. So, it would only be less in particularly compressible examples....and it wasn't even less in this example.

The next assert also fails:
```python
assert plain == brotli.decompress(compressed)
```
`brotli.decompress()` expects bytes; the hexadecimal representation is a string.

Thinking about it....maybe this test was written for Python 2 or something where everything was bytes....except when it wasn't....I've tried to flush that all from memory :)

Reply:
The test was adapted from [the chaquopy version](https://github.com/chaquo/chaquopy/blob/master/server/pypi/packages/brotli/test.py); however, I don't know how recently that test has been run in the chaquopy context. I can't fault your logic about the API itself, though.

Reply:
In the Chaquopy version the hex is only being passed to the `subTest` method so it can appear in failure messages. It isn't being assigned to a local variable.